src/ui/templates/insurance/your-business/turnover.njk

```diff
@@ -75,10 +75,6 @@
             validationError: validationErrors.errorList[annualTurnoverId]
           }) }}
 
-          <p class-"govuk-body">
-            <a class="govuk-link" data-cy="provide-alternative-currency-link" href="{{ PROVIDE_ALTERNATIVE_CURRENCY_URL }}">{{ CONTENT_STRINGS.PROVIDE_ALTERNATIVE_CURRENCY }}</a>
```

Comment:
Note - after other pages have the same treatment ("buyer - outsanding payments", "export contract - agent charges"), we can remove all instances of `provide-alternative-currency-link`.